### PR TITLE
[cxx-interop] Benchmarks for std::span in Swift

### PR DESCRIPTION
**Pull request by susmonteiro**
* std::span is not supported in swift-ci linux
* explicitly pass -std=c++20 in Package.swift: rdar://136138941

### CODE REVIEW

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain Ubuntu 22.04

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain UBI9

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain Ubuntu 22.04

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain UBI9

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci please test

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain UBI9

**Comment by susmonteiro** [comment]
@swift-ci please smoke test

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain UBI9

File: benchmark/Package.swift [hunk 1]
```diff
@@ -129,7 +129,9 @@ targets.append(
     sources: ["main.swift"],
     swiftSettings: [.unsafeFlags(["-cxx-interoperability-mode=default",
                                   "-I",
-                                  "utils/CxxTests"])]))
+                                  "utils/CxxTests",
+                                  // FIXME rdar://136138941
```
Comment (egorzhdan) [inline benchmark/Package.swift:133]: Could you please add a brief one-line description of the issue here in the comment?
That would help folks who don't have access to open rdar links to understand the overall issue.

Comment (susmonteiro) [reply benchmark/Package.swift:133]: Ok sure!

**Comment by susmonteiro** [comment]
@swift-ci please smoke test

**Review by egorzhdan — APPROVED** [review]
LGTM, let's hope it sticks this time 🤞

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci please smoke test

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by susmonteiro** [comment]
@swift-ci Please Build Toolchain Linux platform

**Pull request merged**